core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -481,7 +481,7 @@ oppia.directive('conversationSkin', [
               PlayerPositionService.setActiveCardIndex(numVisibleCards - 1);
               animateToTwoCards(function() {});
             } else if (
-                totalNumCards > 1 &&
+              totalNumCards > 1 &&
                 ExplorationPlayerService.canWindowShowTwoCards() &&
```

Comment:
why is this indented from the previous line, but lines below are not?

Would prefer not to indent at all after && and || if possible. These don't really start new scopes.